### DISCUSSION TITLE
MySQL support?

### DISCUSSION THREAD

**Discussion by BasToTheMax** [pocketbase/pocketbase#1304] [Q&A] [answered]
Is there mysql support?
if not, when will it come?

**Comment by ganigeorgiev** [comment, marked as answer]
No, currently only SQLite is supported (see also https://github.com/pocketbase/pocketbase/discussions/14).

Technically integration with other databases is in the roadmap (_not because of scaling but in order to support providers with ephemeral storage_), but for now this is with the lowest priority and may or not be implemented for v1.

If you haven't already, you can check our roadmap in https://github.com/orgs/pocketbase/projects/2/views/1.
